### PR TITLE
Update the `Running on-cluster` section

### PR DESCRIPTION
**Pull request by haiyanmeng**
Signed-off-by: Haiyan Meng <haiyanmeng@google.com>

### CODE REVIEW

File: docs/addon/walkthrough/README.md [hunk 1]
```diff
@@ -316,80 +329,82 @@ we'll need a Docker image and some manifests.
...
-	          requests:
-	            cpu: 100m
-	            memory: 20Mi
-   EOF
-   ```
+
+```bash
+echo << EOF > config/default/manager_resource_patch.yaml
+apiVersion: apps/v1
+kind: Deployment
```
Comment (haiyanmeng) [inline docs/addon/walkthrough/README.md]: The kind must be changed to `Deployment` here.
The format change makes it possible to copy-and-paste of this snippet.

File: docs/addon/walkthrough/README.md [hunk 2]
```diff
@@ -316,80 +329,82 @@ we'll need a Docker image and some manifests.
...
-
-	# Copy the operator and dependencies into a thin image
-	FROM gcr.io/distroless/static:latest
-	WORKDIR /
-	COPY --from=builder /go/src/sigs.k8s.io/dashboard-operator/manager .
-	COPY --from=kubectl /usr/bin/kubectl /usr/bin/kubectl
-	COPY channels/ channels/
-	ENTRYPOINT ["./manager"]
-   ```
+```Dockerfile
```
Comment (haiyanmeng) [inline docs/addon/walkthrough/README.md]: The format change makes it possible to copy-and-paste of this snippet.

**Comment by johnsonj** [comment]
Awesome, thank you @haiyanmeng !

/lgtm
/approve

**Pull request merged**